tests/clickhouse-test

```diff
@@ -760,6 +771,97 @@ def do_run_tests(jobs, suite, suite_dir, suite_tmp_dir, all_tests, parallel_test
         return num_tests
 
 
+def is_test_from_dir(suite_dir, case):
+    case_file = os.path.join(suite_dir, case)
+    # We could also test for executable files (os.access(case_file, os.X_OK),
+    # but it interferes with 01610_client_spawn_editor.editor, which is invoked
+    # as a query editor in the test, and must be marked as executable.
+    return os.path.isfile(case_file) and any(case_file.endswith(suppotred_ext) for suppotred_ext in TEST_FILE_EXTENSIONS)
+
+
+def removesuffix(text, *suffixes):
+    """
+    Added in python 3.9
+    https://www.python.org/dev/peps/pep-0616/
+
+    This version can work with severtal possible suffixes
```

Comment:
```suggestion
    This version can work with several possible suffixes
```

Reply:
Oops.

Let's commit this change to master.